jujugui/static/gui/src/app/components/deployment/summary/placement/placement.js

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
Note for reviewer: this component is a duplicate of the existing placement component.